frontend/tests/features/cellGuide/cellGuide.test.ts

```diff
@@ -759,11 +759,59 @@ describe("Cell Guide", () => {
         await rowElements[0].locator("td").nth(0).hover();
         await treeIcon.click();
 
-        // check that the eyeClosed button is visible
-        await isElementVisible(
-          page,
-          CELL_GUIDE_CARD_ONTOLOGY_DAG_VIEW_DEACTIVATE_MARKER_GENE_MODE
+        // hover over the node
+        const node = page.getByTestId(
+          `${CELL_GUIDE_CARD_ONTOLOGY_DAG_VIEW_RECT_OR_CIRCLE_PREFIX_ID}-CL:0002319__0-has-children-isTargetNode=false`
+        );
+        await node.hover();
+        await isElementVisible(page, CELL_GUIDE_CARD_ONTOLOGY_DAG_VIEW_TOOLTIP);
+
+        // assert that the tooltip text contains the marker gene information
+        const tooltipText = await page
+          .getByTestId(CELL_GUIDE_CARD_ONTOLOGY_DAG_VIEW_TOOLTIP)
+          .textContent();
+
+        expect(tooltipText).toContain(`${geneSymbol} stats`);
+
+        const legendText = await page
+          .getByTestId(CELL_GUIDE_ONTOLOGY_VIEW_LEGEND_TEST_ID)
+          .textContent();
+        expect(legendText).toContain("Marker Score");
+        expect(legendText).toContain("Expressed in Cells(%)");
+
+        // deactivate marker gene mode and check that the legend and tooltips reverted
+        await page
+          .getByTestId(CELL_GUIDE_CARD_GLOBAL_MARKER_GENE_DROPDOWN)
+          .click();
+        await page.getByRole("option", { name: "No Gene" }).click();
```

Comment:
maybe `NO_GENE_OPTIONS` should be in a `constants.ts` file per @kaloster 's new pattern, so we can import that here instead of hard coding it here